core/editable.js

```diff
@@ -1264,6 +1270,17 @@
 		}
 	} );
 
+	function catchEdgePermissionDenied( callback ) {
+		// Edge randomly throws permission denied when trying to access native elements of detached editor (#3115).
+		try {
+			callback();
+		} catch ( error ) {
+			if ( !CKEDITOR.env.edge || error.number !== -2146828218 ) {
```

Comment:
I think that it's possible but unlikely that error code will change. Yet it is safe, worst thing that could happen is that same error with different code will be thrown. However without using this check we might silent different error.

Previously we did it this way:
https://github.com/ckeditor/ckeditor-dev/blob/c4d250ea5fa0da09781a3ee84c99ca03f09c144b/core/editable.js#L100-L104

But IMO it's not optimal, as it might silent other errors. If error code changes tho, we might repeat logic from above.